package.json

```diff
@@ -117,7 +117,7 @@
     "monaco-editor": "0.15.6",
     "ngtemplate-loader": "2.0.1",
     "node-sass": "4.13.1",
-    "npm": "6.13.4",
+    "npm": "6.13.7",
```

Comment:
Wat? I've never noticed npm in our dependencies. Is this necessary? My gut says we can remove it, unless @torkelo has some background details for introducing it a long time ago (https://github.com/grafana/grafana/blame/010a29a42dd347e382daaf361522d3862b725b95/package.json#L91)

Reply:
Yea, that is strange

Reply:
@dprokop sure; removed.